docs/data/material/components/material-icons/material-icons.md

```diff
@@ -18,24 +18,18 @@ includes the 2,100+ official [Material Icons](https://fonts.google.com/icons?ico
 It depends on `@mui/material`, which requires Emotion packages.
 Use one of the following commands to install it:
 
-:::info
-The `next` tag is used to download the latest <b>pre-release</b>, v6 version. Remove it to get the current stable version.
-:::
-
-<!-- #default-branch-switch -->
```

Comment:
Lets keep this one (same as https://github.com/mui/material-ui/pull/43454/files#r1731548827).